<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewerCollectionLayer</class>
 <widget class="QWidget" name="ViewerCollectionLayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>309</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>285</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>285</width>
    <height>500</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="CollectionLayerCont" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="DictionaryLayerCon2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="addNewCollection">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>53</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>480</width>
          <height>53</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Añadir colección </string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ViewerCollectionItem" name="collectionItem0" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout"/>
       </widget>
      </item>
      <item>
       <widget class="ViewerCollectionItem" name="collectionItem1" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_8"/>
       </widget>
      </item>
      <item>
       <widget class="ViewerCollectionItem" name="collectionItem3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_6"/>
       </widget>
      </item>
      <item>
       <widget class="ViewerCollectionItem" name="collectionItem2" native="true">
        <layout class="QHBoxLayout" name="collectionItem2s"/>
       </widget>
      </item>
      <item>
       <widget class="ViewerVerticalPagerPopup" name="VerticalPagerPopup" native="true">
        <property name="minimumSize">
         <size>
          <width>187</width>
          <height>50</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ViewerVerticalPagerPopup</class>
   <extends>QWidget</extends>
   <header>ViewerVerticalPagerPopup.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ViewerCollectionItem</class>
   <extends>QWidget</extends>
   <header location="global">ViewerCollectionItem.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
